implementations/rust/ockam/ockam_identity/src/lib.rs

```diff
@@ -18,6 +18,12 @@ extern crate core;
 #[macro_use]
 extern crate alloc;
 
+#[cfg(test)]
+extern crate quickcheck;
```

Comment:
Could you please elaborate on why we need this?

Reply:
I got it from there: https://github.com/BurntSushi/quickcheck#the-quickcheck-attribute. On the other hand you're right, `#[macro_use]` is not necessary anymore, I'll remove it

Reply:
And maybe to answer the original question I'm using `quickcheck` [here](https://github.com/build-trust/ockam/pull/4232/files#diff-7ee4043c5dc1f294955fee55a11b5e3abea18e271aed8b94e1fbe7c835880a01R82) for a round-trip test.

Reply:
I'm fine with introducing this dependency, why should it be declared as an extern crate?

Reply:
I was just following the tutorial on the quickcheck repository. I guess they recommend `extern crate` to make the macro attribute available everywhere. I removed those 2 statements and did an import locally instead. Thanks for correcting me on what is idiomatic Rust, I have much to learn in that domain :-).